ee/vellum_ee/workflows/display/nodes/base_node_display.py

```diff
@@ -92,6 +92,29 @@ def _get_explicit_node_display_attr(
         if node_display_attribute is None:
             return None
 
+        origin = get_origin(attribute_type)
```

Comment:
Did this because I was getting this error:
`  def __subclasscheck__(self, cls):
>       raise TypeError("Subscripted generics cannot be used with"
                        " class and instance checks")
E       TypeError: Subscripted generics cannot be used with class and instance checks`